### PR TITLE
Ensure Control Inspection in Diagnostics tool window is triggered on Key Down

### PR DESCRIPTION
**Pull request by emmauss**
## What does the pull request do?
Fixes an issue on Linux where you can't use CTRL + SHIFT to trigger an inspection of a hovered control when the diagnostic tool window is open

## What is the current behavior?
Currently, the key gesture to trigger inspection triggers on KeyDown, but key modifiers aren't set at that point when the modifier is the first key to be pressed.

## How was the solution implemented (if it's not obvious)?
Added extra cases to handled when CTRL or SHIFT is pressed after the other key.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0018219-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public MainWindow()
 
             _keySubscription = InputManager.Instance?.Process
                 .OfType<RawKeyEventArgs>()
-                .Where(x => x.Type == RawKeyEventType.KeyDown)
+                .Where(x => x.Type == RawKeyEventType.KeyUp)
                 .Subscribe(RawKeyDown);
```
[the remark this answers is not in the document]
Comment (emmauss) [reply src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs]: Reverted the event change. I now add extra cases to handle the two modifier press. So it's KeyDown now.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0018227-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0018236-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**